tests/unit/test_tutoring_model.py

```diff
@@ -61,3 +62,108 @@ def test_get_coordinates(
     assert result["KÜ"] == (13, 13, 13, 13)
     assert result["EK"] == (12, 12, 12, 12)
     assert result["LZ"] == (-12, -12, -12, -12)
+
+
+@pytest.mark.parametrize(
+    "learning_style",
+    [
+        (
+            {
+                "id": 1,
+                "characteristic_id": 1,
+                "perception_dimension": "sns",
+                "perception_value": 5,
+                "input_dimension": "vrb",
+                "input_value": 0,
+                "processing_dimension": "act",
+                "processing_value": 9,
+                "understanding_dimension": "seq",
+                "understanding_value": 11,
+            }
+        ),
+        (
+            {
+                "id": 1,
+                "characteristic_id": 1,
+                "perception_dimension": "int",
+                "perception_value": 1,
+                "input_dimension": "vis",
+                "input_value": 11,
+                "processing_dimension": "ref",
+                "processing_value": 1,
+                "understanding_dimension": "glo",
+                "understanding_value": 3,
+            }
+        ),
+    ],
+)
+def test_prepare_les_for_ga(learning_style):
+    list_of_les = []
+    list_of_keys = ["ZF", "KÜ", "SE", "LZ", "ZL", "AN", "ÜB", "EK"]
+    for i, ele_name in enumerate(list_of_keys):
+        le = DM.LearningElement(
+            lms_id=i,
+            activity_type="lesson",
+            classification=ele_name,
+            name="Test LE",
+            university="TH-AB",
+            created_by="Max Mustermann",
+            created_at="2023-09-01",
+        )
+        list_of_les.append(le.serialize())
+    lp = TM.LearningPath(student_id=1, course_id=1, based_on="ga")
+    lp.get_learning_path(
+        student_id=1,
+        learning_style=learning_style,
+        _algorithm="ga",
+        list_of_les=list_of_les,
+    )
+    result = lp.path
+    result = result.split(", ")
```

Comment:
Assert here so you can safely use "split"

Reply:
I added a new Test for ga